mysql-test/suite/rocksdb/t/rpl_row_not_found_rc.cnf

```diff
@@ -0,0 +1,10 @@
+!include suite/rpl/my.cnf
```

Comment:
we typically prefer -master and -slave instead of .cnf

Reply:
Is there any reason behind it?
This will introduce unnecessary differences with the upstream code.